### DISCUSSION TITLE
Opening a new explorer window causes mica to disable

### DISCUSSION THREAD

**Discussion by antaean** [MicaForEveryone/MicaForEveryone#152] [Q&A]
Whenever I open a new window, mica is disabled. Could it be a windows 11 issue?
On the example image, I have mica enabled on the right, and on the left image is a new explorer folder that gets created and doesn't appear to have the effects enabled.

![mica issue](https://user-images.githubusercontent.com/83864273/195777795-b512a2ac-7c74-43a9-94c1-1c35959a8358.png)

Just a slight edit, this appears to happen most often when minimizing then restoring an explorer window

**Comment by general-opex** [comment]
I indentical problem. Fix ?

**Comment by dongle-the-gadget** [comment]
Maybe try [ExplorerFrame](https://github.com/MicaForEveryone/ExplorerFrame/releases)?
